src/mranderson/move.clj

```diff
@@ -292,12 +292,18 @@
   "Replaces all occurrences of the old name with the new name in
   all Clojure source files found in dirs."
   [old-sym new-sym extension dirs watermark]
-  (->> (clojure-source-files dirs extension)
-       (pmap-runner (fn [file]
-                      (->> (str file)
-                           util/file->extension
-                           (update-file file replace-ns-symbol old-sym new-sym watermark extension))))
-       (doall)))
+  (let [files (clojure-source-files dirs extension)
+        dupes (->> (map str files)
+                   frequencies
+                   (filterv #(> (second %) 1)))]
```

Comment:
```suggestion
        dupes (->> files
                   (map str)
                   (frequencies)
                   (filterv #(> (second %) 1)))]
```

Very subjective but this keeps the eyes consistently flowing in the same direction :)

Reply:
Also `map str` could be subject to canonicalization? See other comment

Reply:
On the subjective change... I don't have an objection but wouldn't bother with it.

Reply:
On the absolute file thing... yeah, probably a good idea.

Reply:
On the subjective change... bah, I don't care! I'll commit your suggestion. 🙂